### PR TITLE
For semantic-diff, adding command line flags for old ns/pod lists

### PR DESCRIPTION
**Pull request by shai-doron**

### CODE REVIEW

**Review by adisos — CHANGES_REQUESTED** [review]
The fw-rules tests fail due to some change in the output.
Steps to update the expected output files for fw_rules tests: 
1) rm -rf tests/fw_rules_tests/policies/expected_output/
2) python tests/fw_rules_tests/run_tests.py --all
3) git diff -> verify changes to output are expected and make sense 
4) commit & push the updated expected_output files

File: tests/fw_rules_tests/policies/expected_output/semantic_diff_withIpBlock-scheme_output.txt [hunk 1]
```diff
@@ -26,7 +28,8 @@ src ip block: 172.22.0.0/15 dst_ns: [kube-system] dst_pods: [tier in (frontend)]
 src ip block: 172.31.0.0/16 dst_ns: [kube-system] dst_pods: [tier in (frontend)] conn: UDP 53,
 
 
-np2 and np3 have the same network topology and the same set of policiesnp4 and np4 have the same network topology and the same set of policiesnp1 and np4 are not semantically equivalent.Changed connections between persistent peers:
+np2 and np3 have the same network topology and the same set of policiesnp4 and np4 have the same network topology and the same set of policiesnp1 and np4 are not semantically equivalent.
```
Comment (zivnevo) [inline tests/fw_rules_tests/policies/expected_output/semantic_diff_withIpBlock-scheme_output.txt:31]: This still doesn't look good

**Review by zivnevo — CHANGES_REQUESTED** [review]
Looks good - just one more \n is needed

**Review by zivnevo — APPROVED** [review]
ok to merge

File: tests/fw_rules_tests/policies/expected_output/semantic_diff_withIpBlock-scheme_output.txt (hunk 1 above)
Comment (shai-doron) [reply tests/fw_rules_tests/policies/expected_output/semantic_diff_withIpBlock-scheme_output.txt]: fixed

**Comment by shai-doron** [comment]
@adisos since you have reviewed, you are now also required to approve for merge

**Review by adisos — APPROVED** [review]

**Pull request merged**